db/schema/v002.sql

```diff
@@ -0,0 +1,24 @@
+CREATE INDEX idx_iscn_owner ON iscn (owner); -- 28s, 6.6s
+CREATE INDEX idx_iscn_keywords ON iscn USING GIN (keywords); -- 1.7s, 2.4s
+CREATE INDEX idx_iscn_fingerprints ON iscn USING GIN (fingerprints); -- 27s, 31s
+
+CREATE TABLE iscn_stakeholders (
+  -- pid means primary key ID, which is the auto-increment ID assigned by database in `iscn` table
+  iscn_pid BIGINT REFERENCES iscn (id),
+  -- stakeholder's ID, `sid` so not to be ambiguous with `id` column in `iscn` table
+  sid TEXT,
+  -- stakeholder's name, `sname` so not to be ambiguous with `name` column in `iscn` table
+  sname TEXT
```

Comment:
Cam we record type and reward propotion just kn case later is useful?

Reply:
Currently the `stakeholders` field does not contain these information (only `entity` info), so the migration will need to get these info from the record. I think this will make the migration even slower, not yet tested.

Reply:
Took ~2 mins, not sure if it is really slower, anyway acceptable